js/test/strings-ext/testlistfmt.js

```diff
@@ -2825,7 +2825,7 @@ module.exports.testlistfmt = {
 
         test.expect(2);
         test.ok(fmt !== null);
-        test.equal(fmt.format(["một", "hai"]), "một và hai");
+        test.equal(fmt.format(["một", "hai"]), "một, hai");
```

Comment:
Hold on, if all you did is re-order the data, then there should be not unit test changes. You should investigate why Vietnamese changed....